polyfill/lib/ecmascript.mjs

```diff
@@ -186,9 +186,38 @@ export const ES = ObjectAssign(ObjectAssign({}, ES2019), {
   ToAbsolute: (item) => {
     if (ES.IsAbsolute(item)) return item;
     const isoString = ES.ToString(item);
-    const { zone, ...props } = ES.ParseAbsoluteString(isoString);
-    const datetime = ES.ToDateTime(props, 'reject');
-    return datetime.inTimeZone(zone, 'reject');
+    const {
+      year,
+      month,
+      day,
+      hour,
+      minute,
+      second,
+      millisecond,
+      microsecond,
+      nanosecond,
+      ianaName,
+      offset,
+      zone
+    } = ES.ParseAbsoluteString(isoString);
+    const possibleEpochNs = ES.GetTimeZoneEpochValue(
+      zone,
+      year,
+      month,
+      day,
+      hour,
+      minute,
+      second,
+      millisecond,
+      microsecond,
+      nanosecond
+    );
+    if (possibleEpochNs.length === 1) return new TemporalAbsolute(possibleEpochNs[0]);
```

Comment:
Hmm, now I wonder if we should drop this special case and make sure we verify the offset even if there's only a single result. Or am I missing something?

Reply:
I did it this way to preserve the current behaviour and make only the minimal changes necessary to fix the bug. Maybe this question is part of #428?